README.rst

```diff
@@ -32,6 +32,7 @@ Requirements:
 - pyglet (a bleeding-edge ``tip.zip`` as of the end of May 2014), see the
   "development version" here:
     http://www.pyglet.org/download.html
+- TDTPy (only needed if using Windows)
```

Comment:
Right, will add only if using TDT to readme and warning.

We have our own version of TDTPy?